src/main/java/gregtech/common/tileentities/machines/multi/MTELargeBoiler.java

```diff
@@ -334,6 +334,7 @@ public CheckRecipeResult checkProcessing() {
                             this.mMaxProgresstime += this.excessFuel / 80;
                             this.excessFuel %= 80;
                             this.mEfficiencyIncrease = this.mMaxProgresstime * getEfficiencyIncrease();
+                            this.mMaxProgresstime = (int) (this.mMaxProgresstime * getLongBurntimeRatio(tInput));
                             this.mMaxProgresstime = adjustBurnTimeForConfig(runtimeBoost(this.mMaxProgresstime));
```

Comment:
Small nitpick, I think it would be clearer to read like this
```suggestion
                            int burnTime = (int) (this.mMaxProgresstime * getLongBurntimeRatio(tInput));
                            this.mMaxProgresstime = adjustBurnTimeForConfig(runtimeBoost(burnTime));
```